lib/assets/javascripts/cartodb/table/menu_modules/filters/histogram.js

```diff
@@ -603,18 +589,37 @@
 
     },  
 
+    abbreviateNumber: function (value) {
```

Comment:
Please create a small test suite to check this method will perform well under different values (absence or nullity of values included).